### PR TITLE
feature(Lists): Add List api

### PR DESCRIPTION
**Pull request by geolessel**

### CODE REVIEW

File: test/test_helper.exs [hunk 1]
```diff
@@ -6,5 +6,9 @@ defmodule TestHelper do
     def read(filename) do
       File.read!("test/fixtures/#{filename}")
     end
+
+    def dummy do
+      File.read!("test/fixtures/dummy.json")
```
Comment (jessejanderson) [inline test/test_helper.exs:11]: Does... does this mean I don't need to download json for everything?

File: test/pco_api/people/list_test.exs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    list = List.get(1)
+    assert %PcoApi.Record{id: "1", type: "List"} = list
+  end
+
+  test ".created_by gets the list's creator", %{bypass: bypass} do
+    Bypass.expect bypass, fn conn ->
+      assert "/people/v2/people/1" == conn.request_path
+      Plug.Conn.resp conn, 200, Fixture.read("list.json")
+    end
+    bypass |> record_with_links |> List.created_by
```
Comment (jessejanderson) [inline test/pco_api/people/list_test.exs]: Shouldn't we assert the Record & Type for these?

File: test/test_helper.exs (hunk 1 above)
Comment (geolessel) [inline test/test_helper.exs:11]: yes, but I think for most things it is a good idea still. For example, in my tests, we are testing lists. I still used the list(s) fixtures but dummied out the associations. In those associations, I would use a fixture that matched the association.

Comment (jessejanderson) [inline test/test_helper.exs:11]: Gotcha, yah the annoying ones were the associations anyway, especially with something like Lists where there are a ton.

**Pull request merged**